db/schema.rb

```diff
@@ -33,6 +33,37 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "child_activities", force: :cascade do |t|
+    t.bigint "child_id", null: false
```

Comment:
@goh0222 question: bigint -> is Primary key for active record, is it the same as foreign key for our db table?
![image](https://user-images.githubusercontent.com/94740205/187679833-ef59bb81-389a-4b2f-8389-9228dd3e2123.png)

Reply:
this data type for primary key is auto assigned when we run `rails db:migrate`, so just use it as it is.

Reply:
To answer your question, I think it is.